src/parse_instruction.py

```diff
@@ -334,10 +344,21 @@ def expect(n: str) -> str:
             assert value is None
             word = parse_word(arg_elems)
             maybe_reg = Register(word)
-            if maybe_reg in arch.all_regs:
+            if word in arch.aliased_regs:
+                value = arch.aliased_regs[word]
+            elif maybe_reg in arch.all_regs:
                 value = maybe_reg
             else:
                 value = AsmGlobalSymbol(word)
+        elif tok == '"' and '"' in arg_elems[1:]:
+            # Quoted global symbol
+            # TODO: properly match pairs of quotes, avoid splitting on internal spaces
+            expect('"')
+            symbol = ""
+            while arg_elems[0] != '"':
```

Comment:
```suggestion
            while arg_elems and arg_elems[0] != '"':
```

Reply:
Heh, I had this and then factored it out, because it shouldn't be possible? (We know `arg_elems` contains a `"` from the outer check.)

If you had the same thought, it's probably worth leaving the check to improve readability.

Reply:
I did have the same thought at first because I read this part of the code first. But yeah, I think removing the `'"' in arg_elems[1:]` check is good too, it's weird for a parser to look ahead like that